test/io/perun_test.clj

```diff
@@ -400,6 +425,26 @@ This --- be ___markdown___.")
                          :msg "`pandoc` should parse HTML to markdown"))
 
         (add-txt-file :path "test.md" :content (nth input-strings 0))
+        (p/asciidoctor :out-dir nil
```

Comment:
Then again I couldn't get the tests to fail based on content, so perhaps I'm using the framework incorrectly.